src/core_modules/capture-core/components/ScopeSelector/QuickSelector/OrgUnitSelector.component.js

```diff
@@ -1,185 +1,79 @@
-        if (this.props.selectedOrgUnitId) {
-            return (
-                <div data-test="org-unit-selector">
-                    <Paper square elevation={0} className={this.props.classes.selectedPaper}>
-                        <h4 className={this.props.classes.title}>{ i18n.t('Selected registering unit') }</h4>
-                        <div className={this.props.classes.selectedItemContainer}>
-                            <div>{this.props.selectedOrgUnit.name}</div>
-                            <div className={this.props.classes.selectedItemClear}>
-                                <IconButton data-test="reset-selection-button" className={this.props.classes.selectedButton} onClick={() => this.props.onReset()}>
-                                    <ClearIcon className={this.props.classes.selectedButtonIcon} />
-                                </IconButton>
-                            </div>
-                        </div>
-                    </Paper>
-                    <Dialog
-                        open={this.state.open}
-                        onClose={this.handleClose}
-                    >
-                        <DialogTitle>{i18n.t('Start Again')}</DialogTitle>
-                        <DialogContent>
-                            <DialogContentText>
-                                {i18n.t('Are you sure? All unsaved data will be lost.')}
-                            </DialogContentText>
-                        </DialogContent>
-                        <DialogActions>
-                            <Button
-                                onClick={this.handleClose}
-                                secondary
-                            >
-                                {i18n.t('Cancel')}
-                            </Button>
-                            <Button
-                                onClick={this.handleReset}
-                                primary
-                            >
-                                {i18n.t('Accept')}
-                            </Button>
-                        </DialogActions>
-                    </Dialog>
-                </div>
-            );
-        }
+        const { selectedOrgUnitId, selectedOrgUnit, previousOrgUnitId, onReset, classes } = this.props;
 
         return (
-            <div data-test="org-unit-selector-container">
-                <Paper square elevation={0} className={this.props.classes.paper}>
-                    <h4 className={this.props.classes.title}>{ i18n.t('Registering Organisation Unit') }</h4>
-                    <div>
-                        <OrgUnitField
-                            data-test="org-unit-field"
-                            onSelectClick={this.handleClick}
-                            previousOrgUnitId={this.props.previousOrgUnitId}
-                        />
-                    </div>
-                </Paper>
-            </div>
+            <SelectorBarItem
+                displayOnly={Boolean(selectedOrgUnitId)}
```

Comment:
I tried to remove this line in [my PR](https://github.com/dhis2/capture-app/pull/3169/files#diff-4a059b78a959a9bdf0774576b0a07733742d209f1252ba4a2e3a8609f56094b4L55), but it doesn't seem to have taken effect here. After testing this, the org. unit selector shouldn't be `displayOnly` in any case, I don't think.

Reply:
I must have added it by error when I fixed the merge conflicts 😅 . The last commit fixes it. Thanks!

Reply:
Thanks 👍